extractor/config/pmd/pmd.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<ruleset name="Custom PMD rules"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0
+                             http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+    <description>
+        Custom PMD ruleset for the extractor project
+    </description>
+
+    <rule ref="category/java/bestpractices.xml">
+        <exclude name="GuardLogStatement"/>
+        <exclude name="UnusedPrivateMethod"/>
```

Comment:
Let's use all , and exclude only if it becomes problematic for us.

Reply:
Done